src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -133,38 +146,53 @@ def _get_scripted_model(self, pt_module, example_inputs=None):
         import torch
         import inspect
 
-        def prepare_example_inputs(inputs, input_signature):
-            is_torch_2 = parse(torch.__version__) >= parse("2.0.0")
+        def process_dict_inputs(inputs, input_signature, model):
+            ordered_inputs = []
+            for input_name in input_signature:
+                if input_name in inputs:
+                    ordered_inputs.append(input_name)
+            
+            if ordered_inputs == input_signature[:len(ordered_inputs)]:
+                example_inputs = [inputs[input_name] for input_name in ordered_inputs]
+                if all([isinstance(inp, torch.Tensor) for inp in example_inputs]):
+                    return {"example_inputs": [inputs[name] for name in ordered_inputs]}, ordered_inputs, model
+                return {"example_inputs": example_inputs}, ordered_inputs, model
+
+            input_params = inspect.signature(model.forward if hasattr(model, "forward") else model.__call__).parameters
```

Comment:
can you please make a comment about `input_params` format?

Reply:
added